torch/csrc/jit/passes/tensorexpr_fuser.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <torch/csrc/WindowsTorchApiMacro.h>
+#include <memory>
+
+namespace torch {
+namespace jit {
+
+struct Graph;
+
+// Run TensorExpressions-based fuser.
+TORCH_API void fuseTensorExprs(std::shared_ptr<Graph>& graph);
```

Comment:
Do we need to expose this in the header?  Not that it's a big deal.

Reply:
I think it's good to have it exposed at least for testing purposes.